ddtrace/tracer/tracer_test.go

```diff
@@ -28,12 +28,12 @@ import (
 	"github.com/tinylib/msgp/msgp"
 )
 
-// forceFlush forces a flush of data (traces and services) to the agent
-// synchronously.
-func (t *tracer) forceFlush() {
-	confirm := make(chan struct{})
-	t.flushChan <- confirm
-	<-confirm
+// flushAndWait triggers a flush of the tracer's payload and waits for the transport to
+// receive at least n traces. The wait will time out after 1 second.
+func (t *tracer) flushAndWait(n int) {
+	transport := t.config.transport.(*dummyTransport)
+	t.flushChan <- struct{}{}
+	transport.waitFlush(n, 5*time.Second)
```

Comment:
We should make sure the doc matches this time (1s vs 5s). Is there any reason you went for 5? Seems like a lot, but I don't have any reason against it.

Reply:
1s is fine. I set it to 5 while I was trying to fix a couple tests and neglected to put it back.